analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantNullCheck.CSharp9.cs

```diff
@@ -1,29 +1,163 @@
-﻿object n = 5;
+﻿object a = "";
```

Comment:
General remark - we are missing location verification for new C#9 features. I would test it as well, to have an overview, whether it makes sense.